### ISSUE TITLE
Prisma 4.0.0

### ISSUE THREAD

**Issue by revmischa** [chax-at/transactional-prisma-testing#3]
Can't use Prisma 4:
```
npm ERR! While resolving: platform@1.0.0
npm ERR! Found: @prisma/client@4.0.0
npm ERR! node_modules/@prisma/client
npm ERR!   @prisma/client@"4.0.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @prisma/client@"^2.29.0 || 3" from @chax-at/transactional-prisma-testing@0.4.0
npm ERR! node_modules/@chax-at/transactional-prisma-testing
npm ERR!   dev @chax-at/transactional-prisma-testing@"^0.4.0" from the root project
```

**Comment by Valerionn** [comment]
Thanks for reporting this issue, I have just released a new version (0.5.0) that adds support for Prisma 4 and therefore fixes the dependency resolution problem (Note: You have to manually change the version in your `package.json` to `^0.5.0`, because `^0.4.0` doesn't match the new version)

**Comment by revmischa** [comment]
Amazing, thank you for the super quick response!
